compact/definitions.xml

```diff
@@ -456,13 +456,14 @@ Examples:
     <constant name="ForwardRICHRegion_tan2"        value="Eta3_6_tan * 0.89" />
 
     <constant name="BackwardPIDRegion_zmin"       value="CentralTrackingRegionN_zmax" />
-    <constant name="BackwardPIDRegion_length"     value="25.0*cm" />
+    <constant name="BackwardPIDRegion_length"     value="54.1*cm" />
```

Comment:
I'm going to have to figure out how to make this work with the mRICH. Could be either here (if you have time) or in a separate PR. I couldn't immediately see the `position` element for the detector, but if we position it 'from the outside in' then we probably want position z to be `BackwardPIDRegion_zmax - pfRICH_length / 2.0` or so.

Reply:
I'm not sure how to do this either, need a better understanding of the differences between the two full configurations. It looks like the positions are:
```xml
<constant name="MRICH_zmin"   value="BackwardPIDRegion_zmin"/>
<constant name="PFRICH_zmin"  value="-BackwardRICHRegion_zmin"/>
```

Reply:
Maybe it's time for another `jinja2` template :thinking:

Reply:
Noooooo!

Reply:
resolved after https://github.com/eic/epic/pull/148